modules/virt-creating-data-volumes-using-pvc-api.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-creating-data-volumes-using-pvc-api_{context}"]
+
+= Creating data volumes using the PVC API
+
+When you create a data volume using the *pvc* API, the CDI creates the data volume based on your explicit:
+
+* `accessModes` (`ReadWriteOnce`, `ReadWrtieMany`, `ReadOnlyMany`)
+* `volumeMode` (`Filesystem` or `Block`)
+* `capacity` of `storage` (`5Gi`, for example)
+
+When you explicitly allocate a data volume by using the *pvc* API, you must consider file system overhead. File system overhead is the amount of space required by the CDI to manage the file. Failing to estimate file system overhead results in a storage allocation that is less than you requested.
+
+If you use the *storage* API, the space required for file system overhead is automatically calculated for you, and the amount of space you allocate is exactly what you request.
```

Comment:
If you use the *storage* API, CDI will factor in filesystem overhead and may request a larger PVC to ensure that your disk will fit.

Reply:
We try to stay clear of "may". I get the problem, though in the last words of the sentence :-)

How is:
_If you use the *storage* API, the CDI will factor in filesystem overhead and  request a larger PVC to ensure that your allocation request is successful._